src/components/header/settingsButton/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {TouchableOpacity} from "react-native";
+import Ionicons from "@expo/vector-icons/Ionicons";
+import {styles} from "./styles";
+import {useNavigation} from "@react-navigation/native";
+import {Routes} from "../../../constants";
+
+const SettingsButton = () => {
+    const navigation = useNavigation();
+
+    const handleButtonPress = () => {
+        // FIXME: ###CF### Use correct type
+        // @ts-ignore
+        navigation.navigate(Routes.settings)
+    }
+    return (
+        <TouchableOpacity onPress={handleButtonPress}>
+            <Ionicons name="ios-person-circle" style={styles.icon}/>
+        </TouchableOpacity>
+    )
+}
+
```

Comment:
You can create a **universal button** component and pass the icons there as **children**